### DISCUSSION TITLE
Filter/Skip row(s) based on content?

### DISCUSSION THREAD

**Discussion by OvalMedia** [konnco/filament-import#123] [Q&A] [answered]
Is there a way to filter or skip import of rows based on their content?

Thank you

**Comment by tiagof** [comment]
Yep. Check the documentation on the "Create Record" section.

**Reply by OvalMedia** [reply]
https://github.com/konnco/filament-import#create-record
I don't understand. How do I skip or filter a record based upon certain circumstances. E.g. I want to skip records where the content of a field contains a certain value.

**Reply by tiagof** [reply, marked as answer]
In

```
->handleRecordCreation(function($data){
    return Post::create($data);
})
```

Simply don't create the Post, just return a new instance without creating it in the DB:

```
->handleRecordCreation(function($data){
    return $data['field'] === "ok" ? Post::create($data) : new Post();
})
```

**Reply by OvalMedia** [reply]
My models (laravel 9) don't have a create function but I get the idea. Works now, thx.

**Reply by uqjohart** [reply]
This certainly makes the import continue, but it doesn't update the skipped counter in the notification. 
Any idea how to update that counter?

**Reply by dennyvik** [reply]
> This certainly makes the import continue, but it doesn't update the skipped counter in the notification. Any idea how to update that counter?

Do you found solution for this? not just the skipped counter but also the succeed counter.

**Reply by uqjohart** [reply]
Unfortunately not sorry.
